### DISCUSSION TITLE
Can you work Collaboratively with this tool?

### DISCUSSION THREAD

**Discussion by Gerald-Br** [jgraph/docker-drawio#129] [Q&A] [answered]
I want to work simultaneously with other people on the same diagrams. Is that possible if I host this App on my Server?

**Comment by davidjgraph** [comment, marked as answer]
Not currently, the collab is written specifically for cloudflare workers, it's non-trivial to translate to a self-hosted environment at this time.

**Comment by davidjgraph** [comment]
Duplicate #179
